src/caustics/tests.py

```diff
@@ -10,14 +10,24 @@
 
 __all__ = ["test"]
 
+# Pseudo device as mentioned in https://github.com/pytorch/pytorch/issues/61654
+# using this device, no actual computation is done, but will check
+# for where the tensors are located
+META_DEVICE = torch.device("meta")
```

Comment:
This is very cool! can you use this meta device in the tests here?

Reply:
Yea, I'm planning to use the meta device in a separate PR, since this doesn't really perform computation when used. I figured is a whole separate test of the software.. in this case, just to ensure that Tensors are placed in a single device before compute happens.